src/models/message/ExtensionResponse.ts

```diff
@@ -39,6 +39,13 @@ export interface ExtensionWarningResponse {
 export interface ExtensionStatusResponse {
   action: Action.STATUS_SUCCESS;
 
+  /**
+   * A version number string in SemVer format
+   *
+   * @see https://semver.org/
+   */
+  library: string;
```

Comment:
I would vote for the `Version` suffix for all version fields throughout:

```suggestion
  libraryVersion: string;
```